sklearn_numba_dpex/kmeans/kernels/lloyd_single_step.py

```diff
@@ -203,39 +203,34 @@ def fused_lloyd_single_step(
             # Inner loop: interate on successive windows of size window_n_features
             # that cover all features for current given centroids
             for _1 in range(n_windows_for_features):
-                # centroids_window is modified inplace
                 is_last_feature_window = _1 == last_feature_window_idx
+
+                # centroids_window is modified inplace
                 load_window_of_centroids_and_features(
                     first_feature_idx,
                     loading_centroid_idx,
                     window_loading_centroid_idx,
                     window_loading_feature_offset,
                     current_centroids_t,
                     centroids_window,
```

Comment:
Do you might this PR might be the opportunity to introduce the hint we once talked about during a call?
```suggestion
                load_window_of_centroids_and_features(
                    first_feature_idx,
                    loading_centroid_idx,
                    window_loading_centroid_idx,
                    window_loading_feature_offset,
                    current_centroids_t,
                    # OUT
                    centroids_window,
```

Reply:
I can maybe open a separate PR to add the hint in all kernel calls of the driver file

Reply:
Yes, probably doing it in another PR is more convenient. :+1: